tools/cosmovisor/cmd/cosmovisor/add_upgrade.go

```diff
@@ -33,7 +37,7 @@ func AddUpgrade(cmd *cobra.Command, args []string) error {
 
 	logger := cfg.Logger(os.Stdout)
 
-	upgradeName := args[0]
+	upgradeName := strings.ToLower(args[0])
```

Comment:
strings.ToLower("Δ") = "δ"

Add tests and update documentation?

Reply:
> Send a PR to your branch?
Sure, thanks!